scripts/deps.sh

```
@@ -17,6 +17,9 @@ DEST=/usr/local/bin
 # Install Helm
 curl -SL https://get.helm.sh/helm-${HELM_VERSION}-linux-${ARCH}.tar.gz | tar -xz linux-${ARCH}/helm && mv linux-${ARCH}/helm ${DEST}
 
+# Install Helm v3 as helm3
+curl -SL https://get.helm.sh/helm-${HELM3_VERSION}-linux-${ARCH}.tar.gz | tar -xz linux-${ARCH}/helm && mv linux-${ARCH}/helm ${DEST}/helm3
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)